.github/workflows/swift-analysis.yml

```
@@ -0,0 +1,53 @@
+{\rtf1\ansi\ansicpg1252\cocoartf2822
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 1: did not find expected ',' or '}'

(syntax-check)